runtime/src/lib.rs

```diff
@@ -1621,10 +1612,7 @@ impl_runtime_apis! {
 			list_benchmark!(list,extra, pallet_treasury, Treasury);
 			list_benchmark!(list,extra, pallet_utility, Utility);
 			list_benchmark!(list,extra, pallet_election_provider_multi_phase, ElectionProviderMultiPhase);
-			//TODO: [#463] Currently broken, will be fixed in different issue
-//			list_benchmark!(list,extra,  pdex_migration, PDEXMigration);
-			// list_benchmark!(list,extra,  asset_handler, AssetHandler);
-			let storage_info = AllPalletsWithSystem::storage_info();
+*/			let storage_info = AllPalletsWithSystem::storage_info();
```

Comment:
Whats about all these benchmarks? Why they are still commented out?

Reply:
same as the one below